src/pybind/mgr/dashboard/frontend/src/app/shared/components/usage-bar/usage-bar.component.ts

```diff
@@ -24,30 +25,85 @@ export class UsageBarComponent implements OnChanges {
   calculatePerc = true;
   @Input()
   title = $localize`usage`;
-  @Input()
-  customLegend?: string;
-  @Input()
-  customLegendValue?: string;
-  @Input()
-  showFreeToolTip = true;
-  @Input()
-  showMultisiteTooltip = false;
 
-  usedPercentage: number;
-  freePercentage: number;
+  data: any;
+  options: any;
+  usedPercentage: number = 0;
+  freePercentage: number = 0;
+
+  constructor(private dimless: DimlessPipe, private dimlessBinary: DimlessBinaryPipe) {
+    this.data = [
+      {
+        group: 'Capacity',
+        value: 0
+      }
+    ];
+    this.options = {
+      resizeable: false,
+      title: this.title,
+      meter: {
+        peak: 100,
+        showLabels: false,
+        height: '10px',
+        width: '100px'
+      },
+      tooltip: {
+        enabled: true
+      },
+      color: {
+        scale: {
+          Capacity: '#0043ce'
+        }
+      },
+      height: '80px',
+      width: '200px',
+      toolbar: {
+        enabled: false
+      }
+    };
+  }
 
   ngOnChanges() {
-    if (this.calculatePerc) {
-      this.usedPercentage = this.total > 0 ? (this.used / this.total) * 100 : 0;
-      this.freePercentage = 100 - this.usedPercentage;
+    this.usedPercentage = this.total > 0 ? (this.used / this.total) * 100 : 0;
+    this.freePercentage = 100 - this.usedPercentage;
+    this.data[0].value = Math.round(this.total > 0 ? (this.used / this.total) * 100 : 0);
```

Comment:
What are the initial value for `this.total` ? 
Lets make sure we get correct values in case of null/undefined,